README_LINUX.md

```diff
@@ -318,6 +318,14 @@ names are separated by ':' as in the Unix PATH variable:
    $> export SC_SYNTHDEF_PATH="./synthdefs:/home/sk/SuperCollider/synthdefs"
    ```
 
+Choosing an audio backend
+-------------------------
+
+There are 2 audio backends available in Linux: jack and PulseAudio. **The recommended audio backend on Linux is jack, which is the one that will be built by default.**
```

Comment:
i that true? i'd guess portaudio works on linux, too (and probably is routed through pulseaudio's alsa-emulation into pulseaudio).

Reply:
I mean 2 backends that SuperCollider can use on Linux. I will rephrase it, so that it's clear. Thank you for the feedback!

Reply:
i meant it would be 3 now: jackd, pulseaudio and portaudio - or is portaudio not supported on linux?

Reply:
Ah, ok, I wasn't aware that portaudio was supported by SuperCollider on Linux (I do know it is the one used on Windows), but it might well be the case. Should be easy to update the documentation once I get clarity about this. Thanks!

Reply:
wait - i don't _know_ portaudio is supported on linux, i only _guess_ it is.

Reply:
Ok, after listening to some people, my understanding is that PortAudio is not supported on SuperCollider for Linux (just  experimentally)
But besdeis, PortAudio does not support PulseAudio, which is what we are trying to achieve here, so it is not fit for our purpose.